packages/token-bridge-sdk/src/hooks/useBalance.spec.tsx

```diff
@@ -0,0 +1,147 @@
+/**
```

Comment:
Should we follow the structure we have so far?

```
- hooks
--- __test__
------ useBalance.test.tsx
--- useBalance.ts
```

No real strong preference here, just the way I used it before, inspired by the React repo: https://github.com/facebook/react/tree/main/packages/react/src